Docker/FreshRSS.Apache.conf

```
@@ -14,6 +14,14 @@ TraceEnable Off
 	<IfModule !auth_openidc_module>
 		Error "The auth_openidc_module is not available. Install it or unset environment variable OIDC_ENABLED."
 	</IfModule>
+	Define "Test_OIDC_REMOTE_USER_CLAIM${OIDC_REMOTE_USER_CLAIM}"
+	<IfDefine Test_OIDC_REMOTE_USER_CLAIM>
```

Ah, interesting. I had to look it up to understand the approach :-)

```suggestion
	# https://serverfault.com/questions/1022233/using-ifdefine-with-environment-variables/1022234#1022234
	Define "Test_OIDC_REMOTE_USER_CLAIM${OIDC_REMOTE_USER_CLAIM}"
	<IfDefine Test_OIDC_REMOTE_USER_CLAIM>
```

---

Yeah, that's where I found it! I'll add a comment. It doesn't yet work though, I'm currently debugging to figure out why :slightly_smiling_face: